src/Serilog.Sinks.Fluentd/Serilog.Sinks.Fluentd.csproj

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <Authors>Borys Iermakov</Authors>
     <AssemblyTitle>Serilog.Sinks.Fluentd</AssemblyTitle>
-    <PackageReleaseNotes>0.2.0</PackageReleaseNotes>
+    <PackageReleaseNotes>0.2.3</PackageReleaseNotes>
```

Comment:
@borisermakof I'm not sure if that's the right place to specify the package version since NuGet package has version 0.2.2 and project had 0.2.0. 🤔 Could you please confirm that this change is in fact correct?

Reply:
@igprya I think we should change every occurrence to 0.3.0